lab-geoff/server.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+let http = require('http');
+let url = require('url');
+let querystring = require('querystring');
+let cowsay = require('cowsay');
+let reqHandler = require('./lib/requestHandler.js');
+
+let PORT = process.env.PORT || 3000;
+
+let server = http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  console.log(req.url.path);
+  req.url.query = querystring.parse(req.url.query);
+  console.log(req.url.query);
+
+  if(req.method === 'POST') {
```

Comment:
Should probably validate that there's also a pathname specific to cowsay here in the conditional.